src/Controller/LogVendorController.php

```diff
@@ -30,6 +30,7 @@ public function __construct(
 
     public function indexAction()
     {
+        
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1131)